source/common/Logger.cpp

```diff
@@ -15,8 +15,11 @@ Logger* const Logger::singleton()
 Logger::Logger()
 {
     // Stick with the first output handle we get
-    if (!m_singleton)
```

Comment:
I commented on this earlier. I suggest you use a single `Logger::setSingleton` function and call it like `Logger::setSingleton(new LoggerWhatever);`

Reply:
Done.